docs/install/operator/knative-with-operator-cli.md

```diff
@@ -0,0 +1,220 @@
+# Install by using the Knative Operator CLI Plugin
+
+Knative provides a CLI Plugin to install, configure and manage Knative via the command lines. This CLI plugin facilitates
+you with a parameter-driven way to configure the Knative cluster, without interacting with the complexities of the custom
+resources.
+
+--8<-- "prerequisites.md"
+
+## Install the Knative Operator CLI Plugin
+
+=== "MacOS"
+
+    1. Download the binary `kn-operator-darwin-amd64` for your system from the [release page](https://github.com/knative-sandbox/kn-plugin-operator/releases/tag/knative-v1.6.0).
+
+    1. Rename the binary to `kn-operator`:
+
+        ```bash
+        mv kn-operator-darwin-amd64 kn-operator
+        ```
+
+=== "Linux"
+
+    1. Download the binary `kn-operator-linux-amd64` for your system from the [release page](https://github.com/knative-sandbox/kn-plugin-operator/releases/tag/knative-v1.6.0).
+
+    1. Rename the binary to `kn-operator`:
+
+        ```bash
+        mv kn-operator-linux-amd64 kn-operator
+        ```
+
+Make the plugin executable by running the command:
+
+```bash
+chmod +x kn-operator
+```
+
+Move the file to a directory on your `PATH` by running:
+
+```bash
+mv kn-operator /usr/local/bin
+```
+
+## Verify the installation of the Knative Operator CLI Plugin
+
+You can run the following command to verify the installation:
+
+```bash
+kn-operator -h
```

Comment:
In the past, we had generally opted for usage as a plugin to `kn` rather than as a standalone binary in the docs (i.e `kn operator` vs. `kn-operator`).  At least, that was the case when we added quickstart, not sure if that's still the case.

Reply:
I asked about the same question in the client channel, but no one confirmed that. If we use the plugin based on `kn`, I will change the docs accordingly.